reference/content-types/select.rst

```diff
@@ -62,6 +62,12 @@ Example
 
 You can use symfony expression language to access values from a service.
 
+.. note::
+
+    Be aware the the provided expression is only evaluated during the initial request to the administration interface.
```

Comment:
`the the`. And would highlight the ` ``expression`` `.

Reply:
fixed the `the the` 👍 not sure about the `expression`, because it is not highlighted in the sentence above. but if you want, i can highlibht it 🙂